### ISSUE TITLE
Don't expose port 9103 to Cassandra pods

### ISSUE THREAD

**Issue by Ajith-ceg** [k8ssandra/k8ssandra-operator#1067]
I am having k8s cluster and deploying k8cs with host networking set to true. For my case, host port 9103 is already used by some other process.
So It is not allowing to create Cassandra pods due to port conflict.

The port 9103 is not used for any purpose so it is good to not expose them on the cassandra pods which are being created.

**Comment by burmanm** [comment]
9103 is the MCAC metrics port.

**Comment by Ajith-ceg** [comment]
@burmanm ,that's correct. If we disable the MCAC in the k8ssandracluster yaml, the pod should not expose 9103 port ideally. Because no metrics will be emitted from that port. So , I believe that we no need to expose it in the pod. Correct me if I'm wrong?

**Comment by burmanm** [comment]
Right, in that case (if the disable is used - but not mentioned in the original message), the port should not be exposed. But by default (at least as of 1.9.x) it should be.

**Comment by Ajith-ceg** [comment]
Yep,I should have mentioned it clearly, by which release, this MCAC will be deprecated?
